source/reference/minio-mc/mc-replicate-diff.rst

```diff
@@ -0,0 +1,142 @@
+.. _minio-mc-replicate-diff:
+
+=====================
+``mc replicate diff``
+=====================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc replicate diff
+
+Description
+-----------
+
+.. start-mc-replicate-diff-desc
+
+The :mc:`mc replicate diff` shows a list of ``PUT`` or ``DELETE`` actions to objects and object versions that have not replicated to a remote bucket.
+
+.. end-mc-replicate-diff-desc
+
+You can specify a configured remote target ARN to shows ``PUT`` and ``DELETE`` actions that have not yet replicated to the specific remote target.
```

Comment:
Clarifying question - this only applies for buckets with multiple ARNs? Do we default to something if unspecified?

If so maybe modify to

For buckets with multiple remote targets, you can specify the target ARN to show only objects not replicated to that remote.

Reply:
Deleted this per the next comment.